src/main/webapp/components/timesheet-form/timesheet-form.component.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Creating controller only inside sharedTimesheetApp module,
```

Comment:
Just note at a high level, while components are good to understand, particularly for UI pattern development. I would be sure to study the Module -> Controller -> Service  components of AngularJS to get a clear familiarity with the application architecture you might encounter.